static/js/main.js

```diff
@@ -127,84 +129,36 @@ function goBack(){
     table.innerHTML = tabletext
 }
 
-function getNodes(){
-    var xBeeOBJ
-        xBeeOBJ = {"ID": "2",
-        "Reading": "24",
-        "MinVol": "1",
-        "MaxVol": "200",
-        "MinVal": "40", 
-        "MaxVal": "212", 
-        "Name": "pressure", 
-        "Unit": "Fdeg",
-        "ConnStatus": true}
-        xBeeArray[0] = xBeeOBJ
-        
-        xBeeOBJ = {"ID": "1",
-        "Reading": "22",
-        "MinVol": "0",
-        "MaxVol": "212",
-        "MinVal": "32", 
-        "MaxVal": "100", 
-        "Name": "Temperature", 
-        "Unit": "Fdeg",
-        "ConnStatus": true}
-        xBeeArray[1] = xBeeOBJ
-        
-        xBeeOBJ = {"ID": "4",
-        "Reading": "22",
-        "MinVol": "0",
-        "MaxVol": "212",
-        "MinVal": "32", 
-        "MaxVal": "100", 
-        "Name": "Temperature", 
-        "Unit": "Fdeg",
-        "ConnStatus": true}
-        xBeeArray[2] = xBeeOBJ
-        
-        xBeeOBJ = {"ID": "3",
-        "Reading": "22",
-        "MinVol": "0",
-        "MaxVol": "212",
-        "MinVal": "32", 
-        "MaxVal": "100", 
-        "Name": "Temperature", 
-        "Unit": "Fdeg",
-        "ConnStatus": true}
-        xBeeArray[3] = xBeeOBJ
 
-        xBeeOBJ = {"ID": "3",
-        "Reading": '80',
-        "MinVol": "0",
-        "MaxVol": "212",
-        "MinVal": "32", 
-        "MaxVal": "100", 
-        "Name": "Temperature", 
-        "Unit": "Fdeg",
-        "ConnStatus": true}
-        newxBeeArray[0] = xBeeOBJ
-        
-        xBeeOBJ = {"ID": "4",
-        "Reading": "99",
-        "MinVol": "0",
-        "MaxVol": "212",
-        "MinVal": "32", 
-        "MaxVal": "100", 
-        "Name": "Temperature", 
-        "Unit": "Fdeg",
-        "ConnStatus": true}
-        newxBeeArray[1] = xBeeOBJ
+function getNodes(url){
+    var jsonobj = ''
+    var xhttp = new XMLHttpRequest()
+ 
+    http.onreadystatechange = function(){
+        if(XMLHttpRequest.DONE){
+            jsonobj = JSON.parse(xhttp.responseText)
+            console.log(jsonobj)
+        } 
```

Comment:
This if statement will always be true. You were supposed to replace only the 4, not the entire thing.